### ISSUE TITLE
TextWrapping on TextView with IsReadOnly="true" is broken on Android

### ISSUE THREAD

**Issue by eksperts** [fusetools/fuselibs-public#493]
[Reported on community](https://www.fusetools.com/community/forums/general/selectable_text).

Repro:
```
<App>
  <StackPanel>
    <TextView IsReadOnly="true" Margin="30" FontSize="16" TextWrapping="Wrap" Value="I'm a really long text and I need to be selected and copied! I'm a really long text and I need to be selected and copied! I'm a really long text and I need to be selected and copied! I'm a really long text and I need to be selected and copied!" />
  </StackPanel>
</App>
```
When the `TextView` gets focused (which [should not be possible](https://github.com/fusetools/fuselibs-public/issues/492) in the first place), the wrapped text gets squashed in a single line, as if `TextWrapping="Wrap"` wasn't there.

**Comment by fusebuild** [comment]
The forum thread [Selectable text](https://www.fusetools.com/community/forums/general/selectable_text) posted by Ana Almeida was linked to this issue.

**Comment by kristianhasselknippe** [comment]
i'm pretty sure this gets fixed by pull/500
would you have a chance to test that @eksperts ?

**Comment by kusma** [comment]
Fixed in 1.4
